docs/r2v/data_processing_quick_start.md

```diff
@@ -72,12 +74,13 @@ colormap. The initial alignment can be seen in the following picture:
 By providing slight offsets to correct for the angle of the LiDAR, we can align
 the high intensity points with the road markers:
 
-```bash
-(monodrive-r2v) $ python ./cloud_camera_registration.py -s \
-    --data /path/to/data/collection/kitti \
-    --show-intensity \
-    --lidar_pos_offset 0.0 0.15 0.0 \
-    --lidar_angle_offset -0.5 0.0 0.0
+```
+{
+    "lidar_pos_offset": [0.0, 0.20, -0.06],
+    "lidar_angle_offset": [-0.3, -0.15, 0.0],
+    "gnss_pos_offset": [0.0, 0.0, 0.0],
+    "gnss_angle_offset": [0.0, 0.0, 0.0]
+}
 ```
 
```

Comment:
Can we rephrase this to make it clear the file is already in the correct location?

>Once the values have been determined, place them in the 
`config/sensor_offset.json` file for use in the pipeline.

Reply:
Fixed.